code/modules/client/preference_setup/global/02_settings.dm

```diff
@@ -10,7 +10,7 @@
 	S["motd_hash"]        >> pref.motd_hash
 	S["memo_hash"]        >> pref.memo_hash
 	S["parallax_speed"]   >> pref.parallax_speed
-	S["parallax_toggles"] >> pref.parallax_togs
+	S["parallax_toggles"] >> pref.toggles_secondary
```

Comment:
This key should be renamed to `toggles_secondary`.

Reply:
Resolved.

Reply:
Noot yet. The `S["parallax_toggles"]` should be `S["toggles_secondary"]` like in saving.